### DISCUSSION TITLE
Demo del registro

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#585] [Idee e proposte]
Salve volevo creare una demo del registro dato che me la chiedono in tanti @trinko lei è d'accordo?

**Comment by trinko** [comment]
Si, certo, sarebbe sicuramente essere utile.

**Comment by Davide031** [comment]
Ok allora oggi creero tutto

**Comment by Davide031** [comment]
Ecco il link , per le credenziali 
Contattate fcoco953@gmail.com
 https://Registro2324.altervista.org/public

**Comment by Davide031** [comment]
Link repository https://github.com/Davide031/Demo-Giua-school
